### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by cp-sn** [splunk/security_content#2892]
### **Describe the bug**
Hello,

In ESCU there are some CS with out any "Adaptive Response Actions".

For example [ESCU - Windows Event Log Cleared - Rule] have actions for risk and notable but not the following:
[ESCU - Windows Event For Service Disabled - Rule]
[ESCU - Windows Event Triggered Image File Execution Options Injection - Rule]

Is it expected from the client to create those actions?

### **App Version:**
ESCU 4.14.0

**Comment by ljstella** [comment]
Hi, this is intended.

[ESCU - Windows Event Log Cleared - Rule](https://research.splunk.com/endpoint/ad517544-aff9-4c96-bd99-d6eb43bfbb6a/) is an analytic of type "TTP", while [ESCU - Windows Event for Service Disabled - Rule](https://research.splunk.com/endpoint/9c2620a8-94a1-11ec-b40c-acde48001122/) and [ESCU - Windows Event Triggered Image File Execution Options Injection - Rule](https://research.splunk.com/endpoint/f7abfab9-12ea-44e8-8745-475f9ca6e0a4/) are analytics of type "Hunting". These are two of the possible types, which are most visible in-product from the pre-configured alert actions, but can be seen in this repo or on research.splunk.com on the individual detections pages.

In summary, TTP analytics receive both the Notable and Risk actions, Anomaly analytics receive the Risk Action, Correlation analytics receive the Notable action, and Hunting analytics do not receive one. Hunting searches are written specifically for someone to manually run and review the results as part of threat hunting activities, or to power a dashboard.

For more details about the types, you can check out this page in the [repo's wiki.](https://github.com/splunk/security_content/wiki/Detection-Analytic-Types)
